source/tests/system/nirfmxwlan_driver_api_tests.cpp

```diff
@@ -90,21 +86,18 @@ class NiRFmxWLANDriverApiTests : public Test {
   std::unique_ptr<NiRFmxWLAN::Stub> stub_;
 };
 
-void GetCommaSeparatedStringFromArray(char* array_of_names, int num_of_names, int name_length, char* comma_seperated_name)
+void GetCommaSeparatedStringFromArray(const std::vector<std::string>& array_of_names, std::string& comma_seperated_name)
```

Comment:
nit: the test files are pretty consistent about `snake_case` for helper method names and local variable names. You have some pascal case here and camel case below.

Reply:
I fixed up the PascalCase and camelCase names to be snake_case I found in the file.